README.md

```diff
@@ -8,107 +8,127 @@
 
-<p align="center">
-  <img src="./docs/_images/ablation.png" alt="drawing" width="700"/>
-</p>
+---
+
+## Why NeMo Curator?
 
-NeMo Curator leverages NVIDIA RAPIDS™ libraries like cuDF, cuML, and cuGraph along with Dask to scale workloads across multi-node, multi-GPU environments, significantly reducing data processing time. With NeMo Curator, developers achieve approximately 16× faster fuzzy‑deduplication on an 8 TB RedPajama‑v2 subset, with ~40% lower TCO and near‑linear scaling on 1–4 H100 80 GB nodes. Refer to the chart below to learn more details.
+### Performance at Scale
+
+NeMo Curator leverages NVIDIA RAPIDS™ libraries such as cuDF, cuML, and cuGraph along with Ray to scale workloads across multi-node, multi-GPU environments.
+
+**Proven Results:**
+- **16× faster** fuzzy deduplication on 8 TB RedPajama v2 (1.78 trillion tokens)
+- **40% lower** total cost of ownership (TCO) compared to CPU-based alternatives
+- **Near-linear scaling** from one to four H100 80 GB nodes (2.05 hrs → 0.50 hrs)
 
 <p align="center">
-  <img src="./docs/_images/text-benchmarks.png" alt="drawing" width="700"/>
+  <img src="./docs/_images/text-benchmarks.png" alt="Performance benchmarks showing 16x speed improvement, 40% cost savings, and near-linear scaling" width="700"/>
 </p>
 
-NeMo Curator exhibits near‑linear scaling for fuzzy deduplication. On an 8 TB RedPajama‑v2 subset (~1.78 trillion tokens), processing time drops from 2.05 hours on one H100 80 GB node to 0.50 hours on four nodes. Refer to the scaling chart below to learn more:
+### Quality Improvements
+
+Data curation modules measurably improve model performance. In ablation studies using a 357M-parameter GPT model trained on curated Common Crawl data:
 
 <p align="center">
-  <img src="./docs/_images/scaling.png" alt="drawing" width="700"/>
+  <img src="./docs/_images/ablation.png" alt="Model accuracy improvements across curation pipeline stages" width="700"/>
 </p>
 
-## Contribute to NeMo Curator
+**Results:** Progressive improvements in zero-shot downstream task performance through text cleaning, deduplication, and quality filtering stages.
+
+---
+
+## Learn More
+
+| Resource | Links |
+|----------|-------|
+| **Documentation** | [Main Docs](https://docs.nvidia.com/nemo/curator/latest/) • [API Reference](https://docs.nvidia.com/nemo/curator/latest/apidocs/index.html) • [Concepts](https://docs.nvidia.com/nemo/curator/latest/about/concepts/index.html) |
+| **Tutorials** | [Text](tutorials/text/) • [Image](tutorials/image/) • [Video](tutorials/video/) • [Audio](tutorials/audio/) |
+| **Deployment** | [Installation](https://docs.nvidia.com/nemo/curator/latest/admin/installation.html) • [Infrastructure](https://docs.nvidia.com/nemo/curator/latest/reference/infrastructure/index.html) • [Slurm](https://docs.nvidia.com/nemo/curator/latest/admin/deployment/slurm/image.html) |
```

Comment:
Has the Slurm image example been tested? If not, maybe remove Slurm from the README for now.